TODO

```diff
@@ -129,6 +129,11 @@ Deprecations and removals:
 
 Features:
 
+* refuse using the switch-reboot operation without /etc/initrd-release. Now
```

Comment:
It seems you meant `switch-root`, not `switch-reboot`

Reply:
true. will fix.